CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+
+## Compiling expected taxonomy files
+
+Expected composition data will consist of one of the following types:
+* Microbiome mock community: expected taxonomic composition for a mixture of microbial cells. The taxonomic annotations present in the expected data will be specific to the database version that is used for analysis, and will be meaningless if used for different database versions. Likewise, they may not match the source annotation (i.e., the taxonomy of each strain to the best knowledge of the mock community’s creator) if taxonomic annotations have been revised or if the reference database being used does not contain a given taxonomy.
+* Metagenome mock community: expected gene composition for a mixture of microbial cells/genomes. Gene annotations will be reference database specific, as for microbiome mock communities above.
+
+Other mock community data types are theoretically possible, and could be included in mockrobiota, which only defines required information, files, and file formats. Expected data definitions can expand as other mock community data types are contributed to mockrobiota. If submitting a mock community that is neither marker-gene sequences nor shotgun metagnome sequences, submit an [issue](https://github.com/caporaso-lab/mockrobiota/issues) to confer with the mockrobiota developers before proceeding.
+
+Here we illustrate how expected taxonomy might be compiled from source taxonomy for an example dataset.
+
+Source composition:
+\#Taxonomy | Relative Abundance
+--------- | ------------------
+Staphylococcus aureus ATCC BAA-1718 | 0.200
+Staphylococcus epidermidis ATCC 12228 | 0.200
+Streptococcus agalactiae ATCC BAA-611 | 0.200
+Streptococcus mutans ATCC 700610 | 0.200
+Streptococcus pneumoniae ATCC BAA-334 | 0.200
+
+
+Expected composition, annotated with Greengenes 13_5 reference taxonomy:
+\#Taxonomy | Relative Abundance
+-------- | ------------------
+k\__Bacteria;p\__Firmicutes;c\__Bacilli;o\__Bacillales;f\__Staphylococcaceae;g\__Staphylococcus aureus | 0.200
+k\__Bacteria;p\__Firmicutes;c\__Bacilli;o\__Bacillales;f\__Staphylococcaceae;g\__Staphylococcus;s\__epidermidis | 0.200
+k\__Bacteria;p\__Firmicutes;c\__Bacilli;o\__Lactobacillales;f\__Streptococcaceae;g\__Streptococcus;s\__ | 0.400
+k\__Bacteria;p\__Firmicutes;c\__Bacilli;o\__Lactobacillales;f\__Streptococcaceae;g\__Streptococcus;s\__agalactiae | 0.200
+
+Several issues may arise during database annotation that require careful attention, and hence careful manual curation of expected composition files is important:
+* Specific taxa may not be represented in a reference taxonomy to species level and must be annotated to the nearest common lineage. For example, Streptococcus mutans and Streptococcus pneumoniae are annotated as g\__Streptococcus;s\__ in the example above.
```

Comment:
use italics for Streptococcus mutans, Streptococcus pneumoniae, and g\__Streptococcus;s\__